lib/console/deployments/clusters.ex

```diff
@@ -487,13 +487,18 @@ defmodule Console.Deployments.Clusters do
         %{name: "version", value: cluster.version},
         %{name: "nodePools", value: Jason.encode!(node_pools)}
         | credential_config(cluster)
+        | config_attributes(cluster)
```

Comment:
I think you might need to make this:

` | credential_config(cluster) ++ config_attributes(cluster) ]`